app/test_servo_control/src/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include "mbed.h"
+
+AnalogIn potVoltageIn(PA_0);
+PwmOut servoPwmOut(PA_1);
+
+int main() {
+
+   servoPwmOut.period(0.020); // Seconds
+       
+   while(1) {
+
+       float potVoltage = potVoltageIn.read();
+       servoPwmOut.pulsewidth(0.001 + 0.001*potVoltage/3.3);
```

Comment:
You may have noticed when you were testing this that the servo did not actually go to 180 degrees of range. This is because the accuracy of the servos varies considerably, so the min pulsewidth may be larger or smaller than 1ms, and the maximum may be larger and smaller than 2ms. This takes a bit of testing with the servo to get right, and ive added clarification to the tutorial to explain this.